project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicItemUserAuthenticated.tsx

```diff
@@ -22,7 +22,7 @@ export const MenuIconicItemUserAuthenticated: FC = () => {
     const isHoveredDelayed = useDebounce(isHovered, 200);
 
     const { width } = useGetWindowSize();
-    const isDesktop = width > desktopFirstSizes.tablet;
+    const isDesktop = width > mobileFirstSizes.vl;
```

Comment:
Let's use `useMediaMin` everywhere (if applicable) to unify it.

Reply:
fixed